5.5/root/usr/share/container-scripts/mysql/my-tuning.cnf.template

```diff
@@ -0,0 +1,28 @@
+[mysqld]
+key_buffer_size = ${MYSQL_KEY_BUFFER_SIZE}
+max_allowed_packet = ${MYSQL_MAX_ALLOWED_PACKET}
+table_open_cache = ${MYSQL_TABLE_OPEN_CACHE}
+sort_buffer_size = ${MYSQL_SORT_BUFFER_SIZE}
+read_buffer_size = ${MYSQL_READ_BUFFER_SIZE}
+read_rnd_buffer_size = 256K
+net_buffer_length = 2K
+thread_stack = 256K
+myisam_sort_buffer_size = 2M
+
+# It is recommended that innodb_buffer_pool_size is configured to 50 to 75 percent of system memory.
+innodb_buffer_pool_size = ${MYSQL_INNODB_BUFFER_POOL_SIZE}
+innodb_additional_mem_pool_size = 2M
+# Set .._log_file_size to 25 % of buffer pool size
```

Comment:
Yes, it should be 25% of the buffer pool size that should be 50% of the memory. V2 rounded up 12.5% to 15%.